debian/Dockerfile

```
@@ -0,0 +1,69 @@
+ARG VERSION=""
+
+# Assume this is run from capstone/debian directory
+# Run in the root of the repo
+# docker build -f ./debian/Dockerfile -t packager .
+FROM debian:buster-slim
```

```suggestion
FROM debian:bookworm-slim
```

Just saw it. buster is EOL since 2 years.